sql/updates/latest-dev.sql

```diff
@@ -316,3 +316,6 @@ ALTER TABLE _timescaledb_catalog.dimension
     ADD CONSTRAINT dimension_hypertable_id_fkey FOREIGN KEY (hypertable_id) REFERENCES _timescaledb_catalog.hypertable(id) ON DELETE CASCADE;
 ALTER TABLE _timescaledb_catalog.tablespace
     ADD CONSTRAINT tablespace_hypertable_id_fkey FOREIGN KEY (hypertable_id) REFERENCES _timescaledb_catalog.hypertable(id) ON DELETE CASCADE;
+
+CREATE SCHEMA timescaledb_debug;
+GRANT USAGE ON SCHEMA timescaledb_debug TO PUBLIC;
```

Comment:
Only allow superuser to access objects in this schema.
```suggestion
CREATE SCHEMA timescaledb_debug;
```